assets/js/app.js

```diff
@@ -428,3 +429,32 @@ function setupPlayground() {
 
   $('#inference_submit').on("click", generate);
 }
+
+function setupFormsWithPatchMethod() {
+  $("#creation-form.PATCH").on("submit", function (event) {
+    event.preventDefault();
+
+    var form = $(this);
+    var jsonData = {};
+    form.serializeArray().forEach(function(item) {
+        jsonData[item.name] = item.value;
+    });
+
+    $.ajax({
+        url: form.attr('action'),
+        type: 'PATCH',
+        dataType: "json",
+        data: jsonData,
+        success: function (response, status, xhr) {
+          var redirectUrl = xhr.getResponseHeader('Location');
+          if (redirectUrl) {
+              window.location.href = redirectUrl;
+          }
+        },
+        error: function (xhr, ajaxOptions, thrownError) {
```

Comment:
When it returns due to validation failures, does it display them as regular validation errors below the input field or in an alert popup?

Reply:
It used to display as alert popup. But error under input field is better way to show this, so I changed it to that.